package/cloudshell/iac/terraform/tagging/tag_terraform_resources.py

```diff
@@ -576,7 +577,10 @@ def start_tagging_terraform_resources(main_dir_path: str, logger, tags_dict: dic
         # Error Code 3 mark to the outside tf script (that run me) that there was an error but not because of
         # the override procedure (but because the client tf file has compile errors even before we started the
         # override procedure)
-        exit(3)
+
+        # Had to change exit(3) to "raise" so exception can be handled outside
+        #exit(3)
+        raise ValueError("Exit before the override procedure began because the init/plan failed")
```

Comment:
I suggest the following error message:
"Error during Terraform Init/Plan when applying automated tags"